src/main/resources/static/js/index.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener("DOMContentLoaded", function() {
+    const manageProjectsButton = document.querySelector(".btn-primary");
+
+    manageProjectsButton.addEventListener("click", function(event) {
+        event.preventDefault();
+        alert("Redirecting to the Projects page...");
```

Comment:
This might cause issues later. Must be checked
Won't be smooth to get to another page